app/models/comfy/cms/page.rb

```diff
@@ -82,4 +82,8 @@ def suppress_mirrors_from_links_recirculation
   def ever_been_published?
     revisions.map {|revision| revision.data[:event]}.include?('published')
   end
+
+  def publishable?
+    keywords.map{ |tag| tag.value }.exclude?('do-not-publish')
```

Comment:
i'd prefer to keep the word publish somewhere. so `unpublishable` or `not_publishable` or something alike as opposed to introduce addtional terminology